sdk/build.gradle

```diff
@@ -45,7 +47,11 @@ android {
     }
 
     testOptions {
-        unitTests.returnDefaultValues = true
+        animationsDisabled true
```

Comment:
no = ?

Reply:
Hmmm, looks like I lost it during experimenting with jococo report

BTW if we can run our tests without it, I propose to remove `unitTests.returnDefaultValues` from `testOptions`